components/atoms/forms/Button/button.module.css

```diff
@@ -9,6 +9,7 @@
   border-width: var(--button-border-width);
   border-radius: var(--button-border-radius);
   box-shadow: var(--button-shadow);
+  cursor: pointer;
```

Comment:
Let's leave the cursor optional for now, managed by a prop, like variant, as I believe we have use cases for both variants.